news/2020-11-27-project-loved-november-2020.md

```diff
@@ -0,0 +1,220 @@
+
+[![](/wiki/shared/news/2020-11-27-project-loved-november-2020/catch/3-negaposi-continues.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[sasakure.UK - Negaposi\*Continues](https://osu.ppy.sh/beatmapsets/307364#fruits) by [Badis](https://osu.ppy.sh/users/559308)\
+220 BPM, 2:59 | 6.16★\
+*description written by [Spectator](https://osu.ppy.sh/users/702598)*
+
+Badis' maps have always posed some challenges to players, and this map is one of them. In contrast to modern maps, his maps also actively employ edge dashes and show a pursuit of pattern diversity. Another thing to praise about his map is that unlike other unranked mappers, he always puts effort into the hitsounding. The feedback of the appropriately added hitsounds has a very positive effect on the playability of the map, making it much more enjoyable to play. Not only older players but also newer players will be able to enjoy this map.
+
+[![](/wiki/shared/news/2020-11-27-project-loved-november-2020/catch/4-chocolate.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[otetsu feat. GUMI - Chocolate](https://osu.ppy.sh/beatmapsets/36371#fruits) by [Kuzino](https://osu.ppy.sh/users/158552)\
+180 BPM, 2:02 | 5.28★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+Kuzino is certainly a name that comes to mind when you think about old mappers that made lots of good quality maps. The ones that have been Loved already show that the community definitely enjoys their mapping style and *Chocolate* is no exception to the rule. This is a map that does not require speed like most of today's maps, but rather tests your timing and catcher control. Due to the hyperdash change back in 2012, some of the jumps became more tricky than necessary because of the timing needed to catch them, but it is still very much possible to get a full combo. Both older and newer players can fully enjoy and experience this map, so make sure to give it a shot!
+
+## <a id="mania"></a>osu!mania
+
+osu!mania Loved candidates were chosen by [-mint-](https://osu.ppy.sh/users/8976576), [Abraxos](https://osu.ppy.sh/users/5025064), [lenpai](https://osu.ppy.sh/users/5314573), [Pope Gadget](https://osu.ppy.sh/users/2288341) and [Toaph Daddy](https://osu.ppy.sh/users/7616811)!
```

Comment:
this section doesnt list who wrote the comments. intentional?

Reply:
yes because the mania captains don't specify who picks or writes descriptions